tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -917,7 +909,7 @@ def return_undefined():
 
 def test_zero_dims_fields(cartesian_case):
     if cartesian_case.backend == dace_iterator.run_dace_iterator:
-        pytest.xfail("Not supported in DaCe backend: broadcast")
+        pytest.xfail("Not supported in DaCe backend: zero-dimensional fields")
```

Comment:
Should we enable zero-dimensional fields as well? From running the tests quickly they seem to be treated differently throughout so could also be done separately.

Reply:
If they work, why not enabling them?

Reply:
The don't work!